src/OSPSuite.Core/Domain/ClassificationType.cs

```diff
@@ -7,6 +7,7 @@ public enum ClassificationType
       Simulation,
       Comparison,
       ParameterIdentification,
-      SensitiviyAnalysis
+      SensitiviyAnalysis,
+      QualificationPlan
```

Comment:
Typo: Sensitivi**t**yAnalysis

Reply:
crap!

Reply:
well this is old here. I do not want to change it because this would require project conversion